mock-api/msw/handlers.ts

```diff
@@ -1159,7 +1160,10 @@ export const handlers = makeHandlers({
     const newSubnet: Json<Api.VpcSubnet> = {
       id: uuid(),
       vpc_id: vpc.id,
-      ...body,
+      name: body.name,
+      description: body.description || '',
+      ipv4_block: body.ipv4_block || '',
+      custom_router_id: body.custom_router || '',
```

Comment:
I don't think any of these should fall back to `''` — description and ipv4_block are always strings already and the other one should stay `undefined` if it's `undefined`.